pgpkey/pgpkey.go

```diff
@@ -40,8 +35,11 @@ func (e *IncorrectPassword) Error() string {
 	return fmt.Sprintf("incorrect password: %s", e.decryptErrorMessage)
 }
 
-func Generate(email string, now time.Time) (*PgpKey, error) {
-	return generateKeyOfSize(email, RsaSizeSecureKeyBits, now)
+func Generate(email string, now time.Time, random io.Reader) (*PgpKey, error) {
+	if random == nil {
+		random = cryptorand.Reader
```

Comment:
Ah - here it is!

Reply:
Yeah indeed. That could do with an update to the doc comment